src/app/routes/app-routes.tsx

```diff
@@ -129,7 +129,6 @@ function useAppRoutes() {
             </OnboardingGate>
           }
         >
-          <Route path={RouteUrls.RequestDiagnostics} element={<AllowDiagnosticsModal />} />
```

Comment:
You need to remove the import of `AllowDiagnosticsModal` and also delete that file `@app/pages/onboarding/allow-diagnostics/allow-diagnostics'`